### PR TITLE
Update progress header bar styles in task list

### LINKED ISSUE #32425: Setup experiment: progressive header style tweaks
Follow up to #32298

For the task list progressive header, the following tweaks should be made to the style:

- [ ] Increase progress bar height to `12px`
- [ ] Decrease bottom margin of header to `0`

## Current

Screen Shot 2022-03-30 at 15 29 44

## Desired

Screen Shot 2022-03-30 at 15 30 53

### PR DESCRIPTION
**Pull request by joshuatf**
### All Submissions:

* [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Tweaks the task list progress bar styling.

### Before
Screen Shot 2022-04-05 at 5 07 31 PM

### After

Screen Shot 2022-04-05 at 5 08 27 PM

Closes #32425 

### How to test the changes in this Pull Request:

1.  Make sure that you enable `tasklist-setup-experiment-1` in your feature config
2. Visit the home screen and enable the "two column" layout
3. Note the progress bar style changes are updated as in screenshots above

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### Changelog entry

> Update progress header bar styles in task list

### FOR PR REVIEWER ONLY:

* [ ] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

**Review by louwie17 — DISMISSED** [review]
LGTM 🚀 Nice and straight forward :) 
EDIT: There do seem to be two css related lint errors, I assume this is just spacing?

You might have to generate a changelog for this, although I don't think we really need a changelog, given the progress header is new in this release anyway (assume we already have a changelog for adding a progress header component).

File: plugins/woocommerce-admin/client/task-lists/progress-header/progress-header.scss [hunk 1]
```diff
@@ -19,9 +19,9 @@ $progress-complete-color: #007cba;
 		appearance: none;
 		border: 1px solid #ddd;
 		border-radius: 16px;
-		height: 10px;
+		height: 12px;
 		width: 100%;
-		margin-bottom: 20px;
+		margin-bottom: 0px;
```
Comment (louwie17) [inline plugins/woocommerce-admin/client/task-lists/progress-header/progress-header.scss]: I believe the lint error is probably for `0px` which should be just `0`

**Comment by joshuatf** [comment]
@louwie17 Thanks for the review!  Removed units to fix linting.

Still getting this error even after adding a changelog:
```
Invalid symmetric difference expression 73409592b7c5b5f24e4bead4d26e5857932e9709...3962306d4d5f07427b32a6b045f05faa9406616b
```

Curious if you've seen this before I go down a rabbit hole trying to fix it.

**Comment by louwie17** [comment]
> Still getting this error even after adding a changelog:
> 
> ```
> Invalid symmetric difference expression 73409592b7c5b5f24e4bead4d26e5857932e9709...3962306d4d5f07427b32a6b045f05faa9406616b
> ```
> 
> Curious if you've seen this before I go down a rabbit hole trying to fix it.

I have not run into this particular issue, that feels like an issue with the changelogger script specifically. I know there is an open PR for one bug fix (https://github.com/woocommerce/woocommerce/pull/32525), but that bug doesn't sound like it applies to your PR.
Maybe @jonathansadowski would be of help here?

**Review by louwie17 — APPROVED** [review]
Re-approving this as the lint errors have been fixed 😄

**Comment by jonathansadowski** [comment]
> > Still getting this error even after adding a changelog:
> > ```
> > Invalid symmetric difference expression 73409592b7c5b5f24e4bead4d26e5857932e9709...3962306d4d5f07427b32a6b045f05faa9406616b
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Curious if you've seen this before I go down a rabbit hole trying to fix it.
> 
> I have not run into this particular issue, that feels like an issue with the changelogger script specifically. I know there is an open PR for one bug fix (#32525), but that bug doesn't sound like it applies to your PR. Maybe @jonathansadowski would be of help here?

Sorry for this. I do believe that this is the same issue being discussed in #32525. Until that's merged, please do feel free to ignore the failure if you have a changelog file present.

**Pull request merged**